.github/workflows/create-release.yml

```diff
@@ -130,3 +130,161 @@ jobs:
+
+    outputs:
+      release_id: ${{ steps.create-draft.outputs.release_id }}
+
+  wait-for-prow-jobs:
+    name: Wait for prow jobs
+    needs: [ create-draft, bump-sec-scanners-config ]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Wait for release-kyma-metrics-collector-build status
+        uses: autotelic/action-wait-for-status-check@6556cf50c8fb6608412945382eae73581f56cbb4
+        id: wait-for-post-kmc-release-build
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          statusName: "release-kyma-metrics-collector-build"
+          timeoutSeconds: "600"
+          ref: ${{ needs.bump-sec-scanners-config.outputs.latest_commit}}
+
+      - name: Check if release-kyma-metrics-collector-build status is not success
+        if: steps.wait-for-post-kmc-release-build.outputs.state != 'success'
+        run: |
+          echo 'release-kyma-metrics-collector-buildfailed.'
+          exit 1
+
+  bump-kmc-chart:
+    name: Bump KMC chart
+    needs: [ bump-sec-scanners-config, wait-for-prow-jobs ]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ needs.bump-sec-scanners-config.outputs.latest_commit}}
+      - run: |
+          git checkout main
+          git pull
+
+      - name: Bump KMC chart
+        run: ./hack/scripts/bump_kmc_chart.sh ${{ inputs.name }}
+
+      - name: Create PR if anything changed
+        env:
+          GH_TOKEN: ${{ secrets.BOT_TOKEN }}
+        run: |
+          prs=$(gh pr list -A ${{ env.GIT_NAME }} --state open --json headRefName | jq -r '.[] | .headRefName')
+          if echo $prs | tr " " '\n' | grep -F -q -x ${{ env.BUMP_KMC_BRANCH_NAME }}; then
+            echo "PR already exists, no need to create a new one"
+            echo "PR_NUMBER=$(gh pr list --search "base:main head:${{ env.BUMP_KMC_BRANCH_NAME }}" --json number | jq -r '.[] | .number')" >> $GITHUB_ENV
+          elif [ -z "$(git status --porcelain)" ]; then
+            echo "Nothing changed, no need to create PR"
+            echo "PR_NUMBER=-1" >> $GITHUB_ENV
+          else
+            PR_STATUS=$(./hack/scripts/create_bump_kmc_chart_pr.sh ${{ inputs.name }})
+            echo "PR_NUMBER=$(echo "$PR_STATUS" | tail -n 1)" >> $GITHUB_ENV
+          fi
+
+      - name: Await PR merge
+        timeout-minutes: 45
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          if [ "$PR_NUMBER" -gt 0 ]; then
+            ./hack/scripts/await_pr_merge.sh
+          else
+            echo "Step skipped"
+          fi
+
+      - name: Save latest commit ref
+        id: pull-ref
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git checkout main
+          git stash
+          git pull
+          LATEST_COMMIT=$(git rev-parse HEAD)
+          echo "latest_commit=$LATEST_COMMIT" >> $GITHUB_OUTPUT
+          echo "Latest commit ref $LATEST_COMMIT"
```

Comment:
is this something other than this?
```
git fetch origin
LATEST_COMMIT=$(git rev-parse origin/main)
```

Reply:
I have updated the ref test the behaviour.